.envrc

```diff
@@ -1,14 +1,25 @@
 source activate ruspy_analysis
 
-# We need access to ruspy submodule
-export PYTHONPATH=$PWD/submodules/ruspy
+
+# We need access to robupy submodule
+export PYTHONPATH=$PWD/submodules/robupy
```

Comment:
Please create one block of edits to `PYTHONPATH` and then one applicable comment. All but one are wrong.

Reply:
Adjusted .envrc. I think it doesn't look great. Maybe you have some more feedback. To cite you in another discussion: " Es gibt da schon ein Problembewusstsein :) "